Cargo.toml

```diff
@@ -18,3 +18,7 @@ clap = { version = "4.0.12", features = ["derive"] }
 fuzzy_finder = "0.3.0"
 shellexpand = "2.1.2"
 git2 = "0.14"
+dirs = "4.0"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+promptly = "0.3.1"
```

Comment:
Not really using it now but not sure If I will looks a better solution that my actual local prompt function.

Reply:
Maybe better to add dependencies when they are going to be used?

Reply:
Yes, my reflexion is about if I should use it now or should I postpone that for later 😄

Reply:
I just removed them for now